packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableResizer.ts

```diff
@@ -27,6 +28,11 @@ export default function createTableResizer(
 
     div.style.width = `${TABLE_RESIZER_LENGTH}px`;
     div.style.height = `${TABLE_RESIZER_LENGTH}px`;
+
+    if (zIndex !== undefined) {
+        div.style.zIndex = zIndex.toString();
+    }
```

Comment:
Instead of just pass in a zIndex, I suggest we make it more common so that if in the future people have more requirement to customize the elements, they don't need to add more parameters.

My suggestion is to allow passing in a callback to customized the createElement process. For example:

```ts
class TableResize(
...,
onCreateHelperElements: (createElementData: CreateElementData, helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector') => void) {...}
```
Then in createTableResizer, you can do:

```ts
const data = isRTL ?  KnownCreateElementDataIndex.TableResizerRTL          : KnownCreateElementDataIndex.TableResizerLTR;
onCreateHelperElements?.(data, 'TableResizer');
const div = createElement(data, doc);
```

Then in your code:
```ts
const tabeResize = new TableResize( ..., data => {
   data.style = (data.style || '') + ';zIndex: 10';
});
```

With this way you can do any customization to the helper elements you need.

Reply:
I think it's a good idea, but I'm having trouble coding it due to my lack of knowledge of Rooster codebase & Typescript. In your code sample above `data` is an Index. Maybe you meant to expose `div` in the callback, but I don't know how to properly handle all possible types for this element (looks like it's at least `CreateElementData` and `HTMLDivElement` 🤔 ) and how to correctly declare types for this callback.

You can close this PR if you want, and I'll open an issue so we can track this problem as other users may encounter z-index issues.

Reply:
Ok I can make a PR later.

Reply:
Please see #909

Reply:
Thank you @JiuqingSong  🙏